OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3538,91 +3545,244 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                    boards that were previously fully covered with reddish-brown blind-tooled <material
-                                        key="leather">leather</material>.</decoNote>
+                            <binding xml:id="binding" contemporary="true">
+                                <decoNote xml:id="b1">Two <material key="wood">wooden</material> covered with reddish-brown blind-tooled <material
+                                        key="leather">leather</material>. The front and back boards are decorated with red velvet inlays.</decoNote>
                                 <decoNote xml:id="b2" type="bindingMaterial">
                                     <material key="wood"/>
                                     <material key="leather"/>
                                     <material key="textile"/>
                                     <material key="wood"/>
                                 </decoNote>
-                             
-                          
+                                <decoNote xml:id="b3" type="Cover" color="reddish brown"
+                                    >Blind-tooling decoration on the full leather cover including
+                                    turn-ins, edges, and spine. The decoration was carefully applied
+                                    using different tools.</decoNote>
                                 <decoNote xml:id="b4" type="SewingStations">4</decoNote>
                             </binding>
                         </bindingDesc>
                           
                         
                     </physDesc>
                     
-                <!--    <history>
+                    <history>
                         <origin>
-                            <origDate when="" evidence=""/>
+                            <!-- Dating probably to be revised when taking miniatures into account
+                                (second quarter of 18th century dating by Juel Jensen, probably mainly from analysis of the miniatures),
+                            I cannot restrict the handwriting to such a narrow range-->
+                            The manuscript can be dated palaeographically to the 
+                            <origDate notBefore="1600" notAfter="1799" evidence="lettering">17th-18th century</origDate>. 
+                            Several persons are mentioned in supplication formulas throughout the manuscript:
+                            <persName ref="PRS12911KidanaK"/> and, less often,  <persName ref="PRS12964Gorgoryos"/> and
+                            <persName ref="PRS13026KenfaM"/>.
+           <!--I had created <persName ref="PRS13031WaldaA"/> before looking at the miniatures, he must be mentioned somewhere.
+                            But I did not find him again in the text :(-->
+                                <persName ref="PRS12911KidanaK"/> is also mentioned in the <ref target="#coloph1">colophon</ref> as the commissioner 
```

Comment:
"patron" instead of commissioner?

Reply:
I would stick to "patron" throughout, the common use of "commissioner" is to indicate an authority